engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/StartEventSubscriptionManager.java

```diff
@@ -81,7 +81,7 @@ private void closeMessageExistingStartEventSubscriptions(
                 subscription.getRecord()));
   }
 
-  private DeployedProcess findLastStartProcess(
+  private DeployedProcess hasStartEventMatching(
```

Comment:
@lzgabel I believe you misunderstood my suggestion. I meant to change the parameter name `predicate` to `hasStartEventMatching`